renpy/loadsave.py

```diff
@@ -409,25 +411,23 @@ def save(slotname, extra_info='', mutate_flag=False, include_screenshot=True):
     logf = io.BytesIO()
     try:
         dump((roots, renpy.game.log), logf)
-    except Exception:
-
-        t, e, tb = sys.exc_info()
+    except Exception as e:
 
         if mutate_flag:
-            reraise(t, e, tb)
+            raise e from None
 
         try:
             bad = find_bad_reduction(roots, renpy.game.log)
         except Exception:
-            reraise(t, e, tb)
+            raise e from None
 
         if bad is None:
-            reraise(t, e, tb)
+            raise e from None
 
         if e.args:
-            e.args = (e.args[0] + ' (perhaps {})'.format(bad),) + e.args[1:]
+            e.args = (e.args[0] + f' (perhaps {bad})', *e.args[1:])
 
-        reraise(t, e, tb)
+        raise e from None
```

Comment:
It was these that drew my initial comment, as all these (but for the nested one on line `422`) could be `raise`.

Reply:
Done.